src/main/java/com/dabsquared/gitlabjenkins/GitLabPushTrigger.java

```diff
@@ -177,6 +179,10 @@ public boolean getAcceptMergeRequestOnSuccess() {
         return acceptMergeRequestOnSuccess;
     }
 
+    public boolean getMarkBuildUnstableAsSuccess() {
```

Comment:
According to the JavaBeans Conventions getters for booleans should be prefixed by `is` instead of `get`.